requirements.txt

```diff
@@ -1,8 +1,7 @@
 # Note: This requires users to update the branch to create an envrionment from the branch they are
 #   working on.
-vivarium @ git+https://github.com/ihmeuw/vivarium@release-candidate-spring
-vivarium_public_health @ git+https://github.com/ihmeuw/vivarium_public_health@release-candidate-spring
-# gbd_mapping>=4.0.0
+vivarium @ git+https://github.com/ihmeuw/vivarium@main
```

Comment:
These should be changed in the same way as the `artifact_requirements.txt` file.